doc/index.md

```diff
@@ -57,8 +55,19 @@ The detailed algorithm of spglib is described the following text:
 - Conda: <https://anaconda.org/conda-forge/spglib>
 - PyPI: <https://pypi.org/project/spglib/>
 
+## Contributors
+
+- Atsushi Togo (National Institute for Materials Science)
+- @lan496
+- @LecrisUT
+- [Even more people](https://github.com/spglib/spglib/graphs/contributors)
```

Comment:
Personal preference, I like to put the external contributors front and center for any and all contributions they have
```suggestion
Spglib thrives from the contributions of the community, and we would like to thank each and every one of [the contributors][contributors] as well as everyone participating in the [issue] and [discussion] forums.

The current main developers of Spglib are:
- [Atsushi Togo] (National Institute for Materials Science)
- [Kohei Shinohara]
- [Cristian Le]

[contributors]: https://github.com/spglib/spglib/graphs/contributors
[issue]: https://github.com/spglib/spglib/issues
[discussion]: https://github.com/spglib/spglib/discussions
[Atsushi Togo]: https://github.com/atztogo
[Kohei Shinohara]: https://github.com/lan496
[Cristian Le]: https://github.com/LecrisUT
```

Do we have any honorable developer alumni that we should mention other than those in acknowledgments?

Reply:
Thanks @LecrisUT. My minimum aim was visualization of contributors' name considering academia case, but not limited for it if beneficial, although I have no idea right now. I don't need the line `[Atsushi Togo]: https://github.com/atztogo`, but I don't mean preventing others.

Reply:
> I don't need the line [Atsushi Togo]: https://github.com/atztogo, but I don't mean preventing others.

Btw, that's markdown reference links meaning it is equivalent with:
```markdown
- [Atsushi Togo](https://github.com/atztogo) (National Institute for Materials Science)
```
If you prefer to not have a link, then don't forget to also remove the `[...]` around the reference above.

I would prefer mine to link to my GH profile right now.

> My minimum aim was visualization of contributors' name considering academia case, but not limited for it if beneficial, although I have no idea right now.

Indeed, but I also wanted that part to be a place where other people can find who to contact/credit with regards to the project at the current point in `develop` branch. That's why I've separated the section to be more explicit about the main developers.

Reply:
I'm ok with the current list with my real name.

Reply:
@LecrisUT, I didn't understand markdown format. Now I see it by vscode how you changed. It is nice.